### PR TITLE
PWN-6090 - make a warning screen when address does not have account

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6090

## Description of Work

PWN-6090 - make a warning screen when address does not have account

## Screenshots

Android Emulator - Resizable_API_33:5554 2022-12-12 11-14-24

Android Emulator - Resizable_API_33:5554 2022-12-12 11-13-38

### CODE REVIEW

**Comment by gslevinkov** [comment]
any screen shots please?

**Comment by kabramovp2p** [comment]
оукай

File: app/src/main/java/org/p2p/wallet/newsend/noaccount/SendNoAccountFragment.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            criticalIssue: Boolean
+        ): SendNoAccountFragment = SendNoAccountFragment()
+            .withArgs(
+                ARG_TOKEN_SYMBOL to tokenSymbol,
+                ARG_CRITICAL_ISSUE to criticalIssue
+            )
+    }
+
+    private val tokenSymbol: String by args(ARG_TOKEN_SYMBOL)
+    private val criticalIssue: Boolean by args(ARG_CRITICAL_ISSUE)
```
[the remark this answers is not in the document]
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/newsend/noaccount/SendNoAccountFragment.kt:31]: dknow why not?

File: app/src/main/res/values/send_strings.xml [hunk 2]
```diff
@@ -50,4 +50,11 @@
     <string name="send_switch_to_token">Tap to switch to %s</string>
     <string name="send_enter_amount">Enter Amount</string>
 
+    <string name="send_no_account_title">This address does not have a %s account</string>
+    <string name="send_no_account_critical_message">You will have to pay a one-time fee (~$0.03) to create a %s account for this address</string>
+    <string name="send_no_account_critical_button_text">@android:string/ok</string>
+    <string name="send_no_account_non_critical_message">You will have to pay a one-time fee (~$0.03) to create a %s account for this address. You can choose in which currency to pay with below.</string>
```
[the remark this answers is not in the document]
Comment (kabramovp2p) [reply app/src/main/res/values/send_strings.xml:56]: it is not related to strings it is related to buttons
when you can switch token or not

**Review by davran1001 — APPROVED** [review]

File: app/src/main/res/layout/fragment_send_no_account.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            style="@style/UiKit.Components.Button.Large.Outlined"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/ui_kit_button_large_height"
+            android:layout_marginBottom="8dp"
+            android:backgroundTint="@color/bg_snow"
+            android:text="@string/send_no_account_non_critical_continue"
+            android:textColor="@color/text_night" />
+
+        <org.p2p.uikit.components.UiKitButton
+            android:id="@+id/buttonSwitch"
```
Comment (gslevinkov) [inline app/src/main/res/layout/fragment_send_no_account.xml:87]: buttonSwitch -> buttonSwitchToken

Comment (gslevinkov) [reply app/src/main/res/layout/fragment_send_no_account.xml:87]: can't see click listeners for this button

**Review by gslevinkov — CHANGES_REQUESTED** [review]

**Review by gslevinkov — APPROVED** [review]

**Pull request merged**